minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/util/ConfigTransformer.java

```diff
@@ -77,7 +77,7 @@
 
 public final class ConfigTransformer {
     // Underlying version of NIFI will be using
-    public static final String NIFI_VERSION = "1.2.0";
+    public static final String NIFI_VERSION = "1.4.0";
```

Comment:
Any way this value could reference the `org.apache.nifi.version` value in the pom?

Reply:
There appear to be a couple of different ways[1] but most of them are pretty hacky and not sure it's worth putting in for this one variable. If the variable was logic that was relied on for MiNiFi to work I'd probably lean towards adding it but it's currently only used in one place[2] and that is just for documentation within a file that in general isn't viewed by users (the underlying nifi.properties).

[1] https://stackoverflow.com/questions/11500533/access-maven-properties-defined-in-the-pom
[2] https://github.com/JPercivall/nifi-minifi/blob/baac89b1ddd7a019635bd150c0cce9a37314a371/minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/util/ConfigTransformer.java#L163